client/experimentalclient/client.go

```diff
@@ -1,4 +1,4 @@
-package experimental
+package experimentalclient
```

Comment:
i am going to comment out the init for now. i will investigate it more when we plan to use the operator client.